package-lock.json

```diff
@@ -126,14 +128,15 @@
         "react-resizable-panels": "^2.0.13"
       },
       "peerDependencies": {
-        "@axonivy/ui-icons": "^11.3.0-next",
+        "@axonivy/ui-icons": "~11.3.1-next",
```

Comment:
reason for this problem seem to be the switch from ^ to ~, produces by https://github.com/axonivy/ui-components/commit/71bbdefcbc38bab7d61fe5c318a07f6c900b4c0f